src/status_im/chat/events/commands.cljs

```diff
@@ -0,0 +1,118 @@
+(ns status-im.chat.events.commands
+  (:require [cljs.reader :as reader]
+            [clojure.string :as str]
+            [re-frame.core :refer [reg-fx reg-cofx inject-cofx dispatch trim-v]]
+            [status-im.data-store.messages :as msg-store]
+            [status-im.utils.handlers :refer [register-handler-fx]]
+            [status-im.components.status :as status]
+            [status-im.chat.constants :as const]
+            [status-im.commands.utils :as commands-utils]
+            [status-im.i18n :as i18n]
+            [status-im.utils.platform :as platform]
+            [taoensso.timbre :as log]))
+
+;;;; Helper fns
+
+(defn generate-context
+  "Generates context for jail call"
+  [{:keys [current-account-id chats]} chat-id to]
+  (merge i18n/delimeters
+         {:platform platform/platform
+          :from     current-account-id
+          :to       to
+          :chat     {:chat-id chat-id
+                     :group-chat (get-in chats [chat-id :group-id])}}))
+
+;;;; Coeffects
+
+(reg-cofx
```

Comment:
Yes, some of them definitely will be, like the jail-call effect, I'm going to extract them to common upper level events namespace